app/models/container_deployment_custom_setting.rb

```diff
@@ -0,0 +1,3 @@
+class ContainerDeploymentCustomSetting < ApplicationRecord
+  belongs_to :container_deployment
+end
```

Comment:
Yeah, let's remove this table.  Use the `custom_attributes` table instead.  It has everything you need, and more, it looks.  And, it uses a polymorphic relationship, so you can relate it to any of the containers models.